### PR TITLE
Fix smooth scroll on new collective page

### PR DESCRIPTION
**Pull request by JoyLubega**
Resolve: opencollective/opencollective/issues/2898

# Screenshots

### CODE REVIEW

File: pages/new-collective-page.js [hunk 1]
```diff
@@ -31,10 +31,9 @@ const IncognitoUserCollective = dynamic(
 
 /** Add global style to enable smooth scroll on the page */
 const GlobalStyles = createGlobalStyle`
-  html {
-    scroll-behavior: smooth;
+  html{
```
Comment (Betree) [inline pages/new-collective-page.js]:
```suggestion
  html {
```

File: pages/new-collective-page.js [hunk 2]
```diff
@@ -125,7 +124,7 @@ class NewCollectivePage extends React.Component {
     const collective = data && data.Collective;
     return (
       <Page {...this.getPageMetaData(collective)} withoutGlobalStyles>
-        <GlobalStyles />
+        <GlobalStyles smooth={get(data.Collective, 'path') && true} />
```
Comment (Betree) [inline pages/new-collective-page.js]: It's not a bad workaround, maybe we can write on the `id` instead as it's the only field we'll always keep for sure

```suggestion
        <GlobalStyles smooth={Boolean(get(collective, 'id'))} />
```

**Comment by JoyLubega** [comment]
@Betree  I have used state and Componentdidmount. It works fine but  I am not sure if it is okay for you

**Comment by JoyLubega** [comment]
@Betree  @znarf  Can you help me with some feedback on this PR

**Comment by znarf** [comment]
@JoyLubega Looking at it!

In the meantime, could you rebase to fix the conflicts?

**Comment by znarf** [comment]
Manually tested and this seems to do the job!

Make sure to commit the updated `package-lock.json` and please also update to `5.0.1` which is now the latest version.

**Comment by znarf** [comment]
Looks like the jest snapshots also needs an update now.

**Comment by JoyLubega** [comment]
> Looks like the jest snapshots also needs an update now.

I have update them..

**Comment by JoyLubega** [comment]
@znarf I am done fixing everything

File: package.json [hunk 3]
```diff
@@ -92,10 +92,10 @@
     "react-scrollchor": "6.0.0",
     "react-select": "3.0.8",
     "react-stickynode": "2.1.1",
-    "react-stripe-elements": "6.1.0",
+    "react-stripe-elements": "6.0.1",
+    "react-swipeable": "^5.5.1",
```
Comment (Betree) [inline package.json]: Should stick with a fixed version

```suggestion
    "react-swipeable": "5.5.1",
```

Comment (JoyLubega) [reply package.json]: @Betree  Done.

**Pull request merged**